src/taipy/gui/config.py

```diff
@@ -243,9 +244,20 @@ def _update(self, as_dict: t.Dict[str, t.Any]):
 
         @staticmethod
         def _configure(**properties):
+            """Configure GUI.
+
+            Parameters:
+                **properties (Dict[str, Any]): Keyword arguments that configure `Gui` behavior.<br/>
+                    Please refer to the
+                    [Configuration section](../gui/configuration.md#configuring-the-gui-instance)
+                    of the User Manual for more information on the accepted arguments.
+            Returns:
+                `_GuiSection^`: The gui configuration.
```

Comment:
Private class?

Reply:
it can be easily moved to another file and conditionally imported in the __init__ file

Reply:
I can do it within this PR if both of you agree